changelogs/fragments/5306-add-options-for-authentication.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - znode - possibility to use zookeeper ACL authentication (https://github.com/ansible-collections/community.general/pull/5306).
```

According to https://zookeeper.apache.org/, the correct capitalization is ZooKeeper:
```suggestion
  - znode - possibility to use ZooKeeper ACL authentication (https://github.com/ansible-collections/community.general/pull/5306).
```